nixos/modules/services/backup/borgbackup.nix

```
@@ -555,6 +553,19 @@ in {
             example = "--save-space";
           };
 
+          extraPackages = mkOption {
+            type = with types; listOf package;
+            default = [];
+            defaultText = literalExample "[];";
+            example = literalExample ''
+              with pkgs; [
+                btrfs-progs
+              ]
```

Let's keep it short.
```suggestion
              with pkgs; [ btrfs-progs ]
```

---

I'll add this.